api/tests/integration/infrastructure/repositories/session-repository_test.js

```diff
@@ -23,8 +22,9 @@ describe('Integration | Repository | Session', function() {
         time: '14:30',
         description: 'Première certification EVER !!!'
       });
+      session.id = undefined;
 
-      await databaseBuilder.commit();
+      return databaseBuilder.commit();
```

Comment:
`beforeEach` fait quelque chose de ce qu'on lui retourne ? Il attend la fin de la promesse ?

Reply:
Oui, `beforeEach` attend la résolution, comme `afterEach`, `it`…